joomla/joomla-cms
# [4.0] Wrong order of scripts loading

Pull Request for Issue #32185 .

### Summary of Changes

The fields Media and User have a dependency on the Bootstrap Modal but the order of the execution is wrong. The modal helper function needs to be called before including the field's own script


### Testing Instructions
- Login to the administrator area
- Go to Global Configuration and confirm that there are no console errors
- Edit an article and confirm no console errors and the `Created By`, `Intro Image` and `Full Article Image` fields still working


### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request



### Documentation Changes Required

No
